assets/scss/components/stencil/heroCarousel/_heroCarousel.scss

```diff
@@ -15,7 +15,11 @@
 
 .heroCarousel {
     margin-bottom: (spacing("double") + spacing("single"));
-    margin-top: -(spacing("single") + spacing("base"));
+    margin-top: -(spacing("single"));
+
+    @include breakpoint("medium") {
```

Comment:
I think it would be worth adding a comment explaining _why_ this is necessary (basically copy the explanation David gave on the JIRA). Likewise it's probably worthwhile to add a comment to the `.body` definition to say "If you change the spacing here, please update the heroCarousel too".